pkg/analyze/host_system_packages.go

```diff
@@ -115,33 +119,98 @@ func (a *AnalyzeHostSystemPackages) Analyze(getCollectedFileContents func(string
 	return allResults, nil
 }
 
-func isPackageInstalled(info collect.SystemPackageInfo) bool {
-	if info.Error != "" {
+func getSystemPackageTemplateMap(pkg collect.SystemPackage, osName string, osVersion string) map[string]interface{} {
+	m := map[string]interface{}{
+		"OS":          osName,
+		"OSVersion":   osVersion,
```

Comment:
how about OSVersionMajor so you can do int comparison?

Reply:
i can add that too, but not in place of `OSVersion`

Reply:
agreed